cli/internal/context/context.go

```diff
@@ -29,12 +28,11 @@ const GLOBAL_CACHE_KEY = "the hero we needed"
 
 // Context of the CLI
 type Context struct {
-	RootPackageInfo  *fs.PackageJSON // TODO(gsoltis): should this be included in PackageInfos?
+	// Does not include the root PackageJSON
 	PackageInfos     map[interface{}]*fs.PackageJSON
```

Comment:
fyi, @gsoltis  based on your previous comment

Reply:
Do we still want the TODO to add RootPackageJSON into PackageInfos?

Reply:
I think so, we are possibly going to want it when we do tasks that can run from the root.